sw/host/cryptotest/testvectors/data/schemas/BUILD

```diff
@@ -13,3 +13,8 @@ filegroup(
     name = "ecdsa_sig_ver_schema",
     srcs = ["ecdsa_sig_ver_schema.json"],
 )
+
+filegroup(
```

Comment:
Not blocking: I learned this after I created the other two entries in this file, but you should just be able to use [`exports_files`](https://bazel.build/reference/be/functions#exports_files) here instead of `filegroup` since we're really just looking to access this file alone as a dependency. Then, when you're referencing this file, you can just use `//sw/host/cryptotest/testvectors/data/schemas:kmac_schema.json`

Reply:
Fixed